cmake/modules.cmake

```diff
@@ -67,6 +68,9 @@ if(FFMPEG_FOUND)
   list(APPEND MODULES avformat)
   list(APPEND MODULES swscale)
 endif()
+if(GIO_FOUND)
+  add_subdirectory(modules/ctrl_dbus)
```

Comment:
This should be `list(APPEND MODULES ctrl_dbus)` now, sorry for the inconvenience, but this way we can build modules optional.